packages/context/src/binding-key.ts

```diff
@@ -7,6 +7,7 @@ import {generateUniqueId} from './unique-id';
 
 export type BindingAddress<T = unknown> = string | BindingKey<T>;
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 export class BindingKey<ValueType> {
```

Comment:
Which value is not used here? Is it `ValueType`?

Reply:
Yes. Unused type parameters are checked now by the rule.